dist/bin/kctf-config

```diff
@@ -113,6 +113,9 @@ function kctf_config_create_usage {
   echo -e "                  Possible values are us.gcr.io, asia.gcr.io, and eu.gcr.io" >&2
   echo -e "  --cluster-name  Name of the kubernetes cluster (default: kctf-cluster)" >&2
   echo -e "  --domain-name   Optional domain name to host challenges under" >&2
+  echo -e "  --email-address Optional email address for LetsEncrypt registration (for wildcard certificates)" >&2
+  echo -e "                  To use it, please read and agree to the ACME Subscriber Agreement:" >&2
```

Comment:
what happens if you don't set the email address?

Reply:
You don't get a wildcard certificate, only a ManagedCert

Reply:
This might be interesting to clarify - the LetsEncrypt certificate only helps with reducing the latency, but you still are bound to DNS delays and LoadBalancer delays.

The DNS delays now are the slowest part of the process, but fixing that isn't easily possible unless we reserve IP addresses and assign them to LoadBalancers ahead of time. Reserving an IP costs money, and I'm not sure the extra 1-2 minutes are worth it.

We could make it an option in a future version.